ert3_examples/spe1/README.md

```diff
@@ -13,6 +13,8 @@ that highlights the features of ert3 in a reservoir context.
    suffice in bash.
  - ecl2df: In addition to the ERT dependencies the job `summary2json` utilises
    ecl2df. It can be installed via `pip install ecl2df`.
+ - ert-storage: To store results, ert3 utilizes a service called ert-storage.
+   It can be installed via `pip install ert[storage]`.
```

Comment:
or `pip install ert-storage` ?

Reply:
not to get the version currently compliant with the ert version you are installing